Gemfile

```diff
@@ -16,6 +16,11 @@ end
 ### dep for ci/coverage
 gem 'simplecov', '~> 0.8'
 
-gem 'rubocop', "~> 0.23.0", :platform => [:ruby_19, :ruby_20, :ruby_21]
+# There is no platform :ruby_193 and Rubocop only supports >= 1.9.3
+unless RUBY_VERSION == "1.9.2"
+  gem "rubocop",
+      "~> 0.32.1",
+      :platform => [:ruby_19, :ruby_20, :ruby_21, :ruby_22]
```

Comment:
rspec-dev does not control the Gemfiles. But yes, I am currently in the process of updating rspec-dev with these related changes. We need to add things to the repos directly for now to update rubocop. I plan on pushing a rspec-dev master change after they are all merged.